tests/lax_numpy_reducers_test.py

```diff
@@ -866,5 +866,18 @@ def testCumulativeSumBool(self):
                              dtype=jnp.bool_)
     np.testing.assert_array_equal(np.array([[True], [True], [False]]), out)
 
+  @jtu.sample_product(
+    [dtype for dtype in int_dtypes + unsigned_dtypes],
+  )
+  @jtu.ignore_warning(category=NumpyComplexWarning)
+  def test_low_bitwidth_sum(self, dtype) -> None:
+    rng = rng_factory(self.rng())
```

Comment:
`rng_factory` is undefined here (other tests using this name define it in their parameterization). `jtu.rand_default` is probably fine here.

Reply:
I switched to `jtu.rand_default`